SWDEV-555589 - Edit libdrm source code to create libraries with updated name

raramakr opened the pull request:
librocm_sysdeps_drm and  librocm_sysdeps_drm_amdgpu libraries will be created while building

## Motivation

<!-- The patchelf edited libraries are not providing the symbol version tag. -->

## Technical Details

<!-- Rather than editing libraries to update soname , create libraries with the updated name by editing source code. -->

## Test Plan

<!--Verified  the elfdeps provides field of the libraries. -->

## Test Result

<!-- Required symbild version tags are provided by the libraries -->

## Submission Checklist

- [ ] Look over the contributing guidelines at https://github.com/ROCm/ROCm/blob/develop/CONTRIBUTING.md#pull-requests.

marbre reviewed (dismissed):
Linux builds are failing with e.g.
```
[amd-llvm] : && /opt/rh/gcc-toolset-12/root/usr/bin/c++ -fPIC   -Wl,-Bdynamic -Wl,-z,noexecstack -Wl,/__w/TheRock/TheRock/rocm-systems/projects/rocr-runtime/runtime/hsa-runtime/hsacore.so.link -Wl,--version-script=/__w/TheRock/TheRock/rocm-systems/projects/rocr-runtime/runtime/hsa-runtime/hsacore.so.def -Wl,--enable-new-dtags -shared -Wl,-soname,libhsa-runtime64.so.1 -o rocr/lib/libhsa-runtime64.so.1.18.0 runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/driver/driver.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/driver/kfd/amd_kfd_driver.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/driver/xdna/amd_xdna_driver.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/util/lnx/os_linux.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/util/small_heap.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/util/timer.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/util/flag.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_aie_agent.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_aie_aql_queue.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_blit_kernel.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_blit_sdma.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_cpu_agent.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_gpu_agent.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_hsa_loader.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_aql_queue.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_loader_context.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/hsa_ven_amd_loader.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_memory_region.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_filter_device.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_topology.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/default_signal.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/host_queue.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/hsa.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/hsa_api_trace.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/hsa_ext_amd.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/hsa_ext_interface.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/interrupt_signal.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/intercept_queue.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/ipc_signal.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/isa.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/runtime.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/signal.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/queue.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/cache.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/svm_profiler.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/thunk_loader.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/common/hsa_table_interface.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/loader/executable.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/libamdhsacode/amd_elf_image.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/libamdhsacode/amd_hsa_code_util.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/libamdhsacode/amd_hsa_locks.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/libamdhsacode/amd_options.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/libamdhsacode/amd_hsa_code.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/libamdhsacode/amd_core_dump.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/pcs/hsa_ven_amd_pc_sampling.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/pcs/pcs_runtime.cpp.o  -Wl,-rpath,/__w/TheRock/TheRock/build/third-party/sysdeps/linux/elfutils/build/dist/lib/rocm_sysdeps/lib:/__w/TheRock/TheRock/build/third-party/sysdeps/linux/numactl/build/dist/lib/rocm_sysdeps/lib:  /__w/TheRock/TheRock/build/third-party/sysdeps/linux/elfutils/build/dist/lib/rocm_sysdeps/lib/libelf.so  -ldl  -lpthread  -lrt  libhsakmt/archive/libhsakmt.a  -L/__w/TheRock/TheRock/build/third-party/sysdeps/linux/libdrm/build/stage/lib/rocm_sysdeps/lib/pkgconfig/../../lib  -lrocm_sysdeps_drm  -lrocm_sysdeps_drm_amdgpu  -lpthread  -lrt  -lc  -ldl  /__w/TheRock/TheRock/build/third-party/sysdeps/linux/numactl/build/dist/lib/rocm_sysdeps/lib/libnuma.so  -lrocm_sysdeps_drm && :
[amd-llvm] /opt/rh/gcc-toolset-12/root/usr/libexec/gcc/x86_64-redhat-linux/12/ld: cannot find -lrocm_sysdeps_drm: No such file or directory
[amd-llvm] /opt/rh/gcc-toolset-12/root/usr/libexec/gcc/x86_64-redhat-linux/12/ld: cannot find -lrocm_sysdeps_drm_amdgpu: No such file or directory
[amd-llvm] /opt/rh/gcc-toolset-12/root/usr/libexec/gcc/x86_64-redhat-linux/12/ld: cannot find -lrocm_sysdeps_drm: No such file or directory
[amd-llvm] collect2: error: ld returned 1 exit status
```

raramakr commented:
> Linux builds are failing with e.g.
> 
> ```
> [amd-llvm] : && /opt/rh/gcc-toolset-12/root/usr/bin/c++ -fPIC   -Wl,-Bdynamic -Wl,-z,noexecstack -Wl,/__w/TheRock/TheRock/rocm-systems/projects/rocr-runtime/runtime/hsa-runtime/hsacore.so.link -Wl,--version-script=/__w/TheRock/TheRock/rocm-systems/projects/rocr-runtime/runtime/hsa-runtime/hsacore.so.def -Wl,--enable-new-dtags -shared -Wl,-soname,libhsa-runtime64.so.1 -o rocr/lib/libhsa-runtime64.so.1.18.0 runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/driver/driver.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/driver/kfd/amd_kfd_driver.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/driver/xdna/amd_xdna_driver.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/util/lnx/os_linux.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/util/small_heap.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/util/timer.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/util/flag.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_aie_agent.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_aie_aql_queue.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_blit_kernel.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_blit_sdma.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_cpu_agent.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_gpu_agent.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_hsa_loader.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_aql_queue.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_loader_context.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/hsa_ven_amd_loader.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_memory_region.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_filter_device.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_topology.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/default_signal.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/host_queue.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/hsa.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/hsa_api_trace.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/hsa_ext_amd.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/hsa_ext_interface.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/interrupt_signal.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/intercept_queue.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/ipc_signal.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/isa.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/runtime.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/signal.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/queue.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/cache.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/svm_profiler.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/thunk_loader.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/common/hsa_table_interface.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/loader/executable.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/libamdhsacode/amd_elf_image.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/libamdhsacode/amd_hsa_code_util.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/libamdhsacode/amd_hsa_locks.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/libamdhsacode/amd_options.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/libamdhsacode/amd_hsa_code.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/libamdhsacode/amd_core_dump.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/pcs/hsa_ven_amd_pc_sampling.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/pcs/pcs_runtime.cpp.o  -Wl,-rpath,/__w/TheRock/TheRock/build/third-party/sysdeps/linux/elfutils/build/dist/lib/rocm_sysdeps/lib:/__w/TheRock/TheRock/build/third-party/sysdeps/linux/numactl/build/dist/lib/rocm_sysdeps/lib:  /__w/TheRock/TheRock/build/third-party/sysdeps/linux/elfutils/build/dist/lib/rocm_sysdeps/lib/libelf.so  -ldl  -lpthread  -lrt  libhsakmt/archive/libhsakmt.a  -L/__w/TheRock/TheRock/build/third-party/sysdeps/linux/libdrm/build/stage/lib/rocm_sysdeps/lib/pkgconfig/../../lib  -lrocm_sysdeps_drm  -lrocm_sysdeps_drm_amdgpu  -lpthread  -lrt  -lc  -ldl  /__w/TheRock/TheRock/build/third-party/sysdeps/linux/numactl/build/dist/lib/rocm_sysdeps/lib/libnuma.so  -lrocm_sysdeps_drm && :
> [amd-llvm] /opt/rh/gcc-toolset-12/root/usr/libexec/gcc/x86_64-redhat-linux/12/ld: cannot find -lrocm_sysdeps_drm: No such file or directory
> [amd-llvm] /opt/rh/gcc-toolset-12/root/usr/libexec/gcc/x86_64-redhat-linux/12/ld: cannot find -lrocm_sysdeps_drm_amdgpu: No such file or directory
> [amd-llvm] /opt/rh/gcc-toolset-12/root/usr/libexec/gcc/x86_64-redhat-linux/12/ld: cannot find -lrocm_sysdeps_drm: No such file or directory
> [amd-llvm] collect2: error: ld returned 1 exit status
> ```

Will check and update the patch

raramakr commented:
> > Linux builds are failing with e.g.
> > ```
> > [amd-llvm] : && /opt/rh/gcc-toolset-12/root/usr/bin/c++ -fPIC   -Wl,-Bdynamic -Wl,-z,noexecstack -Wl,/__w/TheRock/TheRock/rocm-systems/projects/rocr-runtime/runtime/hsa-runtime/hsacore.so.link -Wl,--version-script=/__w/TheRock/TheRock/rocm-systems/projects/rocr-runtime/runtime/hsa-runtime/hsacore.so.def -Wl,--enable-new-dtags -shared -Wl,-soname,libhsa-runtime64.so.1 -o rocr/lib/libhsa-runtime64.so.1.18.0 runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/driver/driver.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/driver/kfd/amd_kfd_driver.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/driver/xdna/amd_xdna_driver.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/util/lnx/os_linux.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/util/small_heap.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/util/timer.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/util/flag.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_aie_agent.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_aie_aql_queue.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_blit_kernel.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_blit_sdma.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_cpu_agent.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_gpu_agent.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_hsa_loader.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_aql_queue.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_loader_context.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/hsa_ven_amd_loader.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_memory_region.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_filter_device.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/amd_topology.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/default_signal.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/host_queue.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/hsa.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/hsa_api_trace.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/hsa_ext_amd.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/hsa_ext_interface.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/interrupt_signal.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/intercept_queue.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/ipc_signal.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/isa.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/runtime.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/signal.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/queue.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/cache.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/svm_profiler.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/runtime/thunk_loader.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/core/common/hsa_table_interface.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/loader/executable.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/libamdhsacode/amd_elf_image.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/libamdhsacode/amd_hsa_code_util.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/libamdhsacode/amd_hsa_locks.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/libamdhsacode/amd_options.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/libamdhsacode/amd_hsa_code.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/libamdhsacode/amd_core_dump.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/pcs/hsa_ven_amd_pc_sampling.cpp.o runtime/hsa-runtime/CMakeFiles/hsa-runtime64.dir/pcs/pcs_runtime.cpp.o  -Wl,-rpath,/__w/TheRock/TheRock/build/third-party/sysdeps/linux/elfutils/build/dist/lib/rocm_sysdeps/lib:/__w/TheRock/TheRock/build/third-party/sysdeps/linux/numactl/build/dist/lib/rocm_sysdeps/lib:  /__w/TheRock/TheRock/build/third-party/sysdeps/linux/elfutils/build/dist/lib/rocm_sysdeps/lib/libelf.so  -ldl  -lpthread  -lrt  libhsakmt/archive/libhsakmt.a  -L/__w/TheRock/TheRock/build/third-party/sysdeps/linux/libdrm/build/stage/lib/rocm_sysdeps/lib/pkgconfig/../../lib  -lrocm_sysdeps_drm  -lrocm_sysdeps_drm_amdgpu  -lpthread  -lrt  -lc  -ldl  /__w/TheRock/TheRock/build/third-party/sysdeps/linux/numactl/build/dist/lib/rocm_sysdeps/lib/libnuma.so  -lrocm_sysdeps_drm && :
> > [amd-llvm] /opt/rh/gcc-toolset-12/root/usr/libexec/gcc/x86_64-redhat-linux/12/ld: cannot find -lrocm_sysdeps_drm: No such file or directory
> > [amd-llvm] /opt/rh/gcc-toolset-12/root/usr/libexec/gcc/x86_64-redhat-linux/12/ld: cannot find -lrocm_sysdeps_drm_amdgpu: No such file or directory
> > [amd-llvm] /opt/rh/gcc-toolset-12/root/usr/libexec/gcc/x86_64-redhat-linux/12/ld: cannot find -lrocm_sysdeps_drm: No such file or directory
> > [amd-llvm] collect2: error: ld returned 1 exit status
> > ```
> 
> Will check and update the patch

FYI , With the first patch,  the libraries are created with the updated name librocm_sysdeps_drm.so , librocm_sysdeps_drm.so.1 and librocm_sysdeps_drm.so.1.x
The pkgconfig files are also updated with the name **rocm_sysdeps_drm**
Also the linker library librocm_sysdeps_drm.so is renamed to libdrm.so
ROCR-Runtime use pkgconfig files and its not able to get the linker name , since its renamed to libdrm.so
Updated the patch so that the pkgconfig files still refer to drm
